rootless.md

```diff
@@ -0,0 +1,27 @@
+# Shortcomings of Rootless Podman
+
+* Not a ton of “How To” documentation or documentation in general
+* If you don’t have /etc/subuid and /etc/subgid setup for a user, then it can easily fail
+* This causes issues on machines using Network Based Password information (FreeIPA, Active Directory, LDAP)
```

Comment:
I think you need to clarify what "This" refers to.  I'd suggest either combining this into the prior bullet to make it one  long bullet, or rework this to something like: "If /etc/subuid and /etc/subgid are not setup appropriately, issues will occur on machines...."